flink-python/pyflink/datastream/tests/test_stream_execution_environment.py

```diff
@@ -271,3 +271,16 @@ def test_read_text_file(self):
         results.sort()
         texts.sort()
         self.assertEqual(texts, results)
+
+    def test_execute_async(self):
+        ds = self.env.from_collection([(1, 'Hi', 'Hello'), (2, 'Hello', 'Hi')],
+                                      type_info=Types.ROW(
+                                          [Types.INT(), Types.STRING(), Types.STRING()]))
+        test_sink = DataStreamTestSinkFunction()
+        ds.add_sink(test_sink)
+        job_client = self.env.execute_async("test execute async")
+        job_id = job_client.get_job_id()
+        self.assertIsNotNone(job_id)
+        execution_result = job_client.get_job_execution_result().result()
+        self.assertEqual(str(job_id), str(execution_result.get_job_id()))
+        test_sink.get_results()
```

Comment:
This line is useless.

Reply:
It's used for clearing the collected results maintained in DataStreamTestSinkFunction, otherwise, the following test cases that will use the DataStreamTestSinkFunction will also collect the remnant results, which will cause the result assertion fail.

Reply:
Maybe we can use the clear() method. It just does what we need.